docs/docs/api/index.md

```diff
@@ -0,0 +1,117 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Automate tasks with our API
+
+Our API is designed to be simple and easy to use. It is based on RESTful
+principles and uses standard HTTP methods. This means that you can use the API
+to automate tasks such as creating and managing projects, sessions, and users.
+
+<!-- prettier-ignore -->
+!!! warning
+    We try to keep the API as stable as possible, but we cannot guarantee that
+    it will not change in the future. If you are using the API, make sure to
+    check our [release notes](https://github.com/DSD-DBS/capella-collab-manager/releases)
+    for any breaking changes.
+
+## API Documentation
+
+Please use these links to access the API documentation:
+
+- [:octicons-link-external-16: SwaggerUI](swagger.md) (Interactive SwaggerUI
+  documentation)
+- [:octicons-link-external-16: ReDoc](redoc.md) (Interactive ReDoc
+  documentation)
+- [:octicons-link-external-16: OpenAPI Specification, formerly Swagger Specification](openapi.md)
+  (JSON)
+
+<!-- prettier-ignore -->
+!!! note
+    The API documentation is only available via the documentation
+    of Collaboration Manager instances. Only in this case you'll be able
+    to open the links above. You'll see the API documentation matching the
+    version of the Collaboration Manager instance you are using.
+
+    It is not available on:
+
+    - GitHub Pages
+    - Local non-cluster development environments
+    - ReadTheDocs
+
+## Authentication
+
+To authenticate against the API you can use Personal Access Tokens (PAT).
+
+### PAT Creation
+
+To create a personal access token (PAT) you can go to `Menu` > `Tokens`, insert
+a short description and pick a date until when the token should be valid.
+
+<!-- prettier-ignore -->
+!!! info
+    The token which is generated will disappear after leaving the page.
+    Make sure you save it - you won't be able to access it again.
+
+### PAT Scope
+
+Personal access token have the same scope as the user who created it. It is
+therefore important that you never pass on the token and treat it responsibly.
+All requests are made in the name of the user who issued the token and are
+logged accordingly. If you feel that the token has been lost, revoke it
+immediately and inform the Systems Engineering Toolchain team.
+
+### Revoke a PAT
+
+In order to revoke a token go to `Menu` > `Token`. There you can see a list of
+all tokens that belong to your user. By clicking on the delete button, you can
+delete a token, which will no longer be valid for authentication.
+
+### PAT Usage
+
+You can use the token for basic authentication against the API. Use the token
+as password and your username as username.
```

Comment:
This might be overdone since it's followed immediatley with examples.
"You can use the token in place of a password for basic authenticaiton against the API as in the following examples."